liquibase-standard/src/test/groovy/liquibase/snapshot/jvm/ColumnSnapshotGeneratorTest.groovy

```diff
@@ -49,6 +50,22 @@ class ColumnSnapshotGeneratorTest extends Specification {
         dataType.getTypeName() == "varchar"
     }
 
+    def "ReadDataType specifies column size for mysql datetime data fsp"() {
```

Comment:
I think we can customize a bit this test and make it lightly generic by parameterizing `database`, `columnSize`, and `fps`. Then we can list different scenarios with different values, and test different limits and DBs. At this point, this change is applicable for MySQL and also for MariaDB (which extends from MySQL) and the test result would be the same for either of them, but if there is any future change or any method override it would be nice to be testing the two DBs separately. If you need some help with Spock details we can assist you with it.

Reply:
This would be also nice to do, but it's not mandatory. I can do it if you don't mind.

Thanks,
Daniel.